emails/views.py

```diff
@@ -21,6 +23,7 @@
 
 
 logger = logging.getLogger('events')
+metrics = markus.get_metrics('fx-private-relay')
```

Comment:
`fx-private-relay` is something that the agent should be adding to all metrics emitted by this application. I wouldn't add this here because then it's duplicated in the key names.

Reply:
I see that you're adding this in the `STATSD_PREFIX` already. I think with projects I've worked on, ops adds that in the agent running on the host, so it's not added by the application. My projects aren't designed to run on heroku, though, so maybe it's best to have a `STATSD_PREFIX` and not have that prefix added by the agent running on the host because in Heroku you probably don't have an agent running on the host.

Reply:
We do set some tags, but we don't normally set a prefix for metric names at the agent level, Will. Apps setting a prefix either via their marcus config or via their app code is fine.

For instance, for antenna we set `STATSD_NAMESPACE="socorro.collector"` and then you load that at when initializing the DatadogMetrics bakend, just like Luke is.

Then when you do

```
mymetrics = markus.get_metrics("breakpad_resource")
mymetrics.gauge("work_queue_size", value=len(self.crashmover_queue))
```

We end up with a metric named `socorro.collector.breakpad_resource.work_queue_size`